WalletWasabi.Tests/UnitTests/WabiSabi/Backend/PostRequests/RegisterInputFailureTests.cs

```diff
@@ -332,7 +332,7 @@ public async Task TooMuchFundsAsync()
 		[Fact]
 		public async Task TooMuchVsizeAsync()
 		{
-			WabiSabiConfig cfg = new() { PerAliceVsizeAllocation = 1 };
+			WabiSabiConfig cfg = new() { MaxInputCountByRound = 100_000 };
```

Comment:
maybe a comment to clarify or an `Assert` on the computed per alice allocation, so that this value doesn't seem magic?